packages/key-management/src/cip8/cip30signData.ts

```diff
@@ -39,7 +39,7 @@ export class Cip30DataSignError<InnerError = unknown> extends ComposableError<In
   }
 }
 
-const getAddressBytes = (signWith: Cardano.PaymentAddress | Cardano.RewardAccount | Cardano.DRepID) => {
+export const getAddressBytes = (signWith: Cardano.PaymentAddress | Cardano.RewardAccount | Cardano.DRepID) => {
```

Comment:
We should add some happy path test for each newly exported function, as it becomes a feature

Reply:
How about https://github.com/input-output-hk/cardano-js-sdk/pull/1393/files#diff-5f12d0fee65873887295d29801f8fd2cc5b37b68565182065903ea44c04a7511R21 `isRewardAccount`?

Reply:
A yes, missed that one, done here:

https://github.com/input-output-hk/cardano-js-sdk/pull/1393/commits/ec2231e59a48f40b1f069d30648d77e90ee0efcd